lib/getWorkflow.js

```diff
@@ -16,7 +16,7 @@ const filterNodeName = name =>
  * @return {Array}             List of nodes (jobs)
  */
 const calculateNodes = (jobs) => {
-    const nodes = new Set(['~pr', '~commit', '~release', '~tag']);
+    const nodes = new Set(['~pr', '~commit']);
```

Comment:
Where will we create these nodes if not here?

Reply:
It will be added here: 
https://github.com/sonic-screwdriver-cd/workflow-parser/blob/2243b325c21a19ccdd0a5b3fbeb07ca0bb220d77/lib/getWorkflow.js#L24